Makefile

```diff
@@ -136,7 +138,7 @@ endif
 .PHONY: test
 test: webapp/.npminstall
 ifneq ($(HAS_SERVER),)
-	$(GO) test -race -v ./server/...
```

Comment:
I intentionally removed `-v` here. `-v` is handy with a low number of tests, but with larger plugin it just spams the log. 2/5

OT: Maybe we can add support for `make test -v`.

Reply:
> imagining a CI pipeline as the primary client

The usually case I had in mind was a developer that is testing his/her changes before committing. But the CI case convinced me.